fsharp-backend/src/LibExecution/DvalReprLegacyExternal.fs

```diff
@@ -284,11 +220,15 @@ let toEnduserReadableTextV0 (dval : Dval) : string =
 
   reprfn dval
 
+// SERIALIZER_DEF STJ DvalReprLegacyExternal.toPrettyMachineJsonV1
```

Comment:
This is confusing, as the situation for this is the same as the other functions in this file.

Reply:
I'm really not sure what you mean by this - maybe I've just forgotten the context. I've added an inline TODO to revise the commentary.